.circleci/config.yml

```
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@2.3.0
+  architect: giantswarm/architect@dev:use-architect-with-ct-schema
```

Apply once architect-orb [2.4.1](https://github.com/giantswarm/architect-orb/releases/tag/v2.4.1) has been released

```suggestion
  architect: giantswarm/architect@2.4.1
```

---

Too fast ;)